docusaurus.config.js

```diff
@@ -1,5 +1,5 @@
-const lightCodeTheme = require('prism-react-renderer/themes/github');
-const darkCodeTheme = require('prism-react-renderer/themes/dracula');
+const lightCodeTheme = require('prism-react-renderer/themes/vsLight');
```

Comment:
This change needs to get removed, as we did it in another PR already.  Please rebase to pick it up.

Reply:
I don't see the ```main``` branch having the changes merged yet. I will remove the changes from my PR. I tried to rebase, however everything is already up to date.

Reply:
https://github.com/Seneca-ICTOER/IPC144/blob/main/docusaurus.config.js#L1-L2

Reply:
Ah, I forgot I was on a forked Repo, and had to fetch upstream! Sorry for the confusion.